site/content/entry/citra-progress-report-2018-Q1.md

```diff
@@ -0,0 +1,233 @@
+    title="Sooo many servers to choose from!" >}}
+
+Since its initial release in Canary, developers have listened to the community and tried very hard
+to fix bugs affecting many games which made them incompatible. They have also added other features
+to make the multiplayer experience richer.
+
+### Compatible Titles
+We are proud to announce that multiplayer currently works in the following list of titles:
+
+* Dragon Quest Monsters: Joker 3 Professional
+* Luigi's Mansion: Dark Moon
+* Mario Party: The Top 100
+* Monster Hunter 3G & 3 Ultimate
+* Monster Hunter 4, 4G & 4 Ultimate
+* Monster Hunter X
+* Monster Hunter XX
+* New Super Mario Bros. 2
+* Pokémon Omega Ruby / Alpha Sapphire
+* Pokémon Sun / Moon
+* Pokémon Ultra Sun / Ultra Moon
+* Pokémon Virtual Console
+* Super Smash Bros.
+* The Legend of Zelda: Tri Force Heroes
+
+### Incompatible Titles
+For various reasons, the following games were tested and do not work:
+
+* Asphalt Assault 3D
+* Code of Princess (*a fix is in works*)
+* Dragon Quest Monsters: Terry's Wonderland 3D
+* F1 2011
+* Kirby Fighters Deluxe
+* Kirby Triple Deluxe
+* Mario Party Island Tour
+* Mario Party Star Rush
+* Mario Kart 7
+* Planet Crashers
+* Resident Evil: The Mercenaries 3D
+* Ridge Racer 3D
+* Sonic Generations
+* Street Fighter IV
+* Tetris Ultimate
+
+***Also note that Download Play and Spotpass titles do not work as they are beyond the scope of current
+local multiplayer***.
+
+**Note**: Although multiplayer is supported, some games might still experience issues with some games.
+Since the servers are located in various locations, latency will play a major role in your gameplay
+experience, so always choose a server that is nearby (geographically). Please use our [support forums](https://community.citra-emu.org/),
+[IRC](http://webchat.freenode.net/?channels=citra), or [Discord](https://citra-emu.org/discord/) for
+help regarding issues with multiplayer.
+
+Read more about the multiplayer feature update [here](https://citra-emu.org/entry/announcing-networking-support/).
+
+## citra-qt: Hardware Shader Improvements (post GLvtx) by [wwylele](https://github.com/wwylele) and [degasus](https://github.com/degasus)
+
+Citra has come a long way from just being able to render graphics in 2015, to being able to run games
+at a playable state. Still, Citra was only fast for users who had better hardware. Users with slower
+processors suffered from lag and stutters during gameplay.
+
+Now, thanks to the combined efforts of [phantom](https://github.com/phanto-m), [jroweboy](https://github.com/jroweboy), [wwylele](https://github.com/wwylele), [MerryMage](https://github.com/MerryMage), and many more, we
+now have a near-complete hardware renderer with huge improvements. Citra is now able to make full use
+of the host GPU and render graphics by utilizing the GPU power. [wwylele](https://github.com/wwylele) and [MerryMage](https://github.com/MerryMage) took it upon themselves to simplify the humongous heap of
```

Comment:
Leftover from my initial review: This could be phrased better, more like "Citra now uses the host GPU to its fullest to render graphics."